packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -604,11 +606,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         };
         const chunks = await tryFetchBlob<[string, string[]][]>(chunksBlobName) ?? [];
         const metadata = await tryFetchBlob<IContainerRuntimeMetadata>(metadataBlobName);
+        const electedSummarizer = await tryFetchBlob<ISerializedElection>(electedSummarizerBlobName);
```

Comment:
do we want a new blob here? would it make sense in metadata? is it fine due to aggregation?

Reply:
This is a good question. I've done it both ways for now (in a separate commit).. open to suggestions. @vladsud